crates/forge-runner/src/running.rs

```diff
@@ -198,20 +198,17 @@ pub fn run_test_case(
     runner_config: &Arc<RunnerConfig>,
     runner_params: &Arc<RunnerParams>,
 ) -> Result<RunResultWithInfo> {
+    // we need to do this "trick" to be compatible with the older scarb versions
     ensure!(
-        case.available_gas.is_none(),
-        "\n    Attribute `available_gas` is not supported\n"
+        case.available_gas != Some(0),
+        "\n    Make sure you use the scarb version where `available_gas` attribute is supported\n"
```

Comment:
I would specify the version, it will be supported from 2.5.0 onwards (I assume we want to merge both related PRs before 2.5.0 which will be released next week, right?)